packages/big-design/src/components/Form/Description/spec.tsx

```diff
@@ -17,6 +17,7 @@ test('renders description with link', () => {
         text: 'more',
         href: 'f',
         target: 't',
+        external: true,
```

Comment:
This will not show an external link icon unless `target="_blank"`. Was this intended for this addition?